network-store-server/src/main/resources/db/changelog/changesets/migrationCgmesControlArea_20250415T120000Z.sql

```diff
@@ -0,0 +1,40 @@
+--  set area
+INSERT INTO area(networkuuid, variantnum, id, name, areatype, voltagelevelids, interchangetarget, fictitious,
```

Comment:
Should only be migrated for variantnum=0 ?
For exemple for a study we created with the root node and the first and that we migrated, we want areas and areaboundaries only in variantnum=0 I think

Reply:
well I think it does not exist on other variant so it wont change a thing but I can do that

Reply:
CgmesControlAreas are on Network. So they are present on each variant. After migration, we want the Area in the root variant only

Reply:
done